airbyte-integrations/connectors/destination-clickhouse-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -27,24 +27,34 @@
         "examples": ["8123"],
         "order": 1
       },
+      "tcp-port": {
```

Comment:
Is that also part of the [verify that a destination is able to write any ISO8601-compliant date string] ticket?